### PR TITLE
Adding OpenSUSE install instructions

### PR DESCRIPTION
**Pull request by mnavarro83**
Verified working on Leap 15. Should be fine on Tumbleweed as well but not independently verified.

### CODE REVIEW

File: src/installation/install-linux-opensuse.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+```
+
+## 9 And finally with one additional flag to make it persist reboots
+
+```bash
+    sudo firewall-cmd --add-port=4000/udp --permanent
+    sudo firewall-cmd --add-port=4000/tcp --permanent
+```
+
+That's it. Navigate to <printer_ip>:4000 (example: "192.168.100.10:4000") in your browser and have a day!
```
Comment (NotExpectedYet) [inline src/installation/install-linux-opensuse.md:79]: Should this be "have a nice day!" ?

**Review by NotExpectedYet — CHANGES_REQUESTED** [review]
Looks good, just one question.

**Comment by mnavarro83** [comment]
Private joke of mine I suppose. I find it puts a lot of pressure on people to put a qualifier on their day; it is enough for me that they are having a day. I am happy to change it if it is confusing or out of place, just let me know!

Michael

File: src/installation/install-linux-opensuse.md (hunk 1 above)
Comment (mnavarro83) [reply src/installation/install-linux-opensuse.md:79]: Private joke of mine I suppose. I find it puts a lot of pressure on people to put a qualifier on their day; it is enough for me that they are having a day. I am happy to change it if it is confusing or out of place, just let me know!

Michael

Comment (NotExpectedYet) [reply src/installation/install-linux-opensuse.md:79]: No I'm laughing about it now ahaha I'll keep it :) consider it accepted so I'll merge and start build :) big thanks again.

**Review by NotExpectedYet — APPROVED** [review]

**Pull request merged**